### ISSUE TITLE
"Customizing drop down items with ComponentRenderer" demo source is not displayed

### ISSUE THREAD

**Issue by Haprog** [vaadin/vaadin-combo-box-flow#353]
See https://vaadin.com/components/vaadin-combo-box/java-examples/presentation

The demo "Customizing drop down items with ComponentRenderer" does not show the source.

Problem is here: https://github.com/vaadin/vaadin-combo-box-flow/blob/master/vaadin-combo-box-flow-demo/src/main/java/com/vaadin/flow/component/combobox/demo/ComboBoxView.java#L316-L317

This should be a one liner that matches the string in the [addCard()](https://github.com/vaadin/vaadin-combo-box-flow/blob/master/vaadin-combo-box-flow-demo/src/main/java/com/vaadin/flow/component/combobox/demo/ComboBoxView.java#L349) call.

So it should be:
```java
        // source-example-heading: Customizing drop down items with ComponentRenderer
```
instead of the current:
```java
        // source-example-heading: Customizing drop down items with
        // ComponentRenderer
```

This was probably caused by automatic formatting and that's probably why some of the other demos in the same file wrap the demo source example in:
```java
//@formatter:off
...
//@formatter:on
```

**Comment by Haprog** [comment]
This is a regression since https://github.com/vaadin/vaadin-combo-box-flow/pull/293

**Comment by anasmi** [comment]
In 3.0 fixed by https://github.com/vaadin/vaadin-combo-box-flow/pull/356

In 3.1 will be fixed by https://github.com/vaadin/vaadin-combo-box-flow/pull/356

**Comment by Haprog** [comment]
Closing as fixed by #356 (not sure why merging it didn't automatically close this)
